webapp/sockets/TcpSocket.js

```diff
@@ -214,11 +225,12 @@ var TcpSocket = function(io) {
         dataSentFlags[instance.id] = serviceFlag;
 
         TcpManager.connect(instance).then(function() {
+          serviceFlag.isDataSent = false;
+          TcpManager.updateService(instance);
```

Comment:
Remember that it is temporary change. When the STATUS services updates [711](https://trac.dpi.inpe.br/terrama2/ticket/711), it should be removed when ticket is closed.